src/components/layout/SharedLayout/SharedLayout.jsx

```diff
@@ -1,12 +1,13 @@
 import { Outlet } from "react-router";
 import styles from "./SharedLayout.module.css";
-import { Header, Footer, Loader } from "../../../components";
+import { Header, Hero, Footer, Loader } from "../../../components";
 import { Suspense } from "react";
 
 const SharedLayout = () => {
   return (
     <div className={styles.container}>
       <Header />
+      <Hero />

```

Comment:
The hero section should only appear on the home page; it shouldn't be included in the main layout.

Reply:
+